packages/lesswrong/components/localGroups/GroupLinks.tsx

```diff
@@ -89,21 +129,36 @@ const GroupLinks = ({ document, classes }: {
       <div className={classes.groupLinks}>
         {document.facebookLink
           && <Tooltip
-            title="Link to Facebook Group"
+            title={`Link to Facebook ${isEvent ? 'Event' : 'Group'}`}
             placement="top-end"
           >
-            <a href={document.facebookLink}><IconButton className={classes.iconButton} color="inherit">
-              <FacebookIcon className={classes.facebookIcon}/>
-            </IconButton></a>
```

Comment:
I don't think we should be nesting buttons inside of links so I removed the buttons here